packages/generator/test/generated/esm-imports-async/util.expected.ts

```diff
@@ -0,0 +1,26 @@
+// This file was auto-generated by 'typesafe-i18n'. Any manual changes will be overwritten.
+/* eslint-disable */
+
+import { i18nString as initI18nString, i18nObjectLoaderAsync } from 'typesafe-i18n'
+import type { Translation, TranslationFunctions, Formatters, Locales } from './types.actual.js'
```

Comment:
I think for type only imports the `.js` file extension is not needed

Reply:
Ok, I'll remove it, but we maybe have to let a notice saying to not mix type imports with normal imports. This could break...